store/storetest/user_store.go

```diff
@@ -3357,7 +3357,7 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 
 	// Add u2 to team 1
 	store.Must(ss.Team().SaveMember(&model.TeamMember{TeamId: teamId, UserId: u2.Id}, -1))
-	u2.UpdateAt = store.Must(ss.User().UpdateUpdateAt(u2.Id)).(int64)
```

Comment:
Still missing there the `require.Nil(t, err)` to replace properly the `store.Must` behavior. (Here and in all the other `store.Must` usages).